system/jaeger-operator/values.yaml

```diff
@@ -0,0 +1,14 @@
+# Default values for jaeger.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+#
+global:
+  region: DEFINED_IN_VALUES_FILE
```

Comment:
This is an anti-pattern. Please don't define default-values that are not usable. Rather comment it out.

Reply:
I added this to fix the linting error

> executing "jaeger/templates/openstack-jaeger.yaml" at <.Values.global.region>: nil pointer evaluating interface {}.region

how to fix it otherwise?

Reply:
Where do you get that error? In a local `helm template` run? In the automatic helm-charts tests? In your attempt to deploy the chart?

Reply:
the error is from the pull request validation. https://ci.eu-de-2.cloud.sap/builds/396813

Reply:
Ok, then you need to provide an extra values file for the CI to use for validation. [Here](https://github.com/sapcc/helm-charts/blob/master/openstack/nova/ci/test-values.yaml) is an example from nova. It uses any `.yaml` file in the chart's `ci` subdirectory.

Reply:
thx, fixed by values in ci/